Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -294,7 +295,10 @@ private bool CanReadAll(string resource)
                                       // our API.
                 case Resource.ADVISOR:
                     // User is admin
-                    if (user_groups.Contains(AuthGroup.SiteAdmin))
+                    if (user_groups.Contains(AuthGroup.AcademicInfoView))
+                        return true;
+                    // User looks its own profile
```

Comment:
We should refer to users as either "his or her" or "their" but probably not "its".

Reply:
Right! I will change it.

Also, it seems like line 297 and 298 have some conflicts, which makes me cannot merge this PR. Should I delete one of them?

Reply:
What do you mean by conflicts exactly? If you are referring to the merge conflicts that GitHub is warning you about, that just means you need to merge or rebase develop into your current branch.

You can do so by running `git rebase origin develop` or `git merge origin develop` on the command line. Address the merge conflicts that the command line notifies you about, commit, then push changes to GitHub. I'm happy to walk you through it during office hours tomorrow morning if you would prefer.

Reply:
Okay! I tried but it didn't seem right. I guess I'll see you in office hour.😂 By the way, as usual, I have a class at 9:45, so if you have time, can we meet a little earlier?